test/paper_trail_test.rb

```diff
@@ -16,6 +16,19 @@ class PaperTrailTest < ActiveSupport::TestCase
     assert PaperTrail.enabled?
   end
 
+  context 'can be disabled' do
+    setup {PaperTrail.config.disable!}
+
+    should 'be disabled' do
+      Thread.new do
+        assert !PaperTrail.enabled?
+        PaperTrail.enabled = true
+        assert PaperTrail.enabled?
```

Comment:
Given that the return value is actually boolean (not merely truthy) please use `assert_equal`, not `assert`. (`assert` tests truthiness)